client/src/pages/AdminPage/ApprovalList.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+
+function ApprovalList() {
+  const [userApproved, setUserApproved] = useState(false);
+  const [userDenied, setUserDenied] = useState(false);
+
+  const handleApproved = (value) => {
+    // userApproved ||
+    //   fetch(`/userApproved`, {
+    //     method: 'PUT',
+    //     body: JSON.stringify({content}),
+    //     headers: {"Content-Type": "application/json"}
+    //     })
+    //     .then(() => setUserApproved(true))
+    //     .catch(err => console.log(err, 'err PUT approved'));
+    if (value) {
+      console.log('hit')
+      setUserApproved(true)
+    }
+  };
+
+  // const handleDeny = () => {
+  //   userDeny ||
+  //     fetch(`/userDenied`, {
+  //       method: 'PUT',
+  //       body: JSON.stringify({content}),
+  //       headers: {"Content-Type": "application/json"}
+  //       })
+  //       .then(() => setUserDenied(true))
+  //       .catch(err => console.log(err, 'err PUT deny'));
+  // };
+
```

Comment:
@positivefx Im sure you're already on it but I think it would be a good idea to define the fetches in our api folder.